src/page.js

```diff
@@ -1,4 +1,4 @@
-function appPage(creator, storeSave, storeDelete, storeSet) {
+function appPage(creator, storeSave, storeDelete, storeSet, refreshValues) {
```

Comment:
WDYT of changing these args to something like this:
```suggestion
function appPage(creator, {storeSave, storeDelete, storeSet, storeRefresh}) {
```

Reply:
doni